stories/documentation/navigation/vertical-navigation/vertical-navigation-basic.stories.ts

```diff
@@ -0,0 +1,54 @@
+import { Meta, Story } from '@storybook/angular';
+
+interface VerticalNavigationBasicStory {
+	disabled: boolean;
+}
+
+export default {
+	title: 'Documentation/Navigation/VerticalNavigation/Basic',
+	argTypes: {
+		disabled: {
+			control: {
+				type: 'boolean',
+			},
+		},
+	},
+} as Meta;
+
+function getTemplate(args: VerticalNavigationBasicStory): string {
+	const disabled = args.disabled ? `disabled tabindex="-1"` : '';
+	return `<nav class="verticalNavigation">
+	<ul class="verticalNavigation-list">
+		<li class="verticalNavigation-list-item" aria-expanded="false">
+			<a href="#" class="verticalNavigation-list-item-link" ${disabled}>Item <span aria-hidden="true" class="lucca-icon icon-southArrow"></span></a>
```

Comment:
I think this should be a `button` and I don't think we can use the disabled attribute on `a` tags.

Reply:
@MarcFairbrother I also created a <button> version on QA page. I'm thinking to provide both `[disabled]` and `.is-disabled` option 🤔

Reply:
Ok, I don't think we should have `<a disabled>` in the docs though as it's a bad practice.

Reply:
Ah sorry I misunderstood it was specific expandable actions. fixing it 👍